src/components/ListView.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
You can also import `{ Component } ` here so then in the class signature you can just extend `Component` without the dot.